modules/machineset-modifying.adoc

```diff
@@ -1,22 +1,23 @@
 // Module included in the following assemblies:
 //
-// 
+//
 // * machine_management/modifying-machineset.adoc
+// * scalability_and_performance/recommended-cluster-scaling-practices.adoc
 
 [id="machineset-modifying_{context}"]
 = Modifying a MachineSet
 
-To make changes to a MachineSet, edit the MachineSet YAML. Then remove all machines associated with the MachineSet 
-by deleting or scaling down the machines to `0`. Then scale the replicas back to the desired number. 
-Changes you make to a MachineSet do not affect existing machines. 
+To make changes to a MachineSet, edit the MachineSet YAML. Then remove all machines associated with the MachineSet
+by deleting or scaling down the machines to `0`. Then scale the replicas back to the desired number.
```

Comment:
'by deleting each machine' would be better phrasing here.  Otherwise it sounds like it might imply deleting the machineSet, which is bad.